spec/models/user_spec.rb

```diff
@@ -54,4 +54,29 @@
     @user.contacts << contact
     expect(@user.contacts).to include(contact)
   end
+
+  it "adds user to contact request list of other user" do
+    contact = FactoryBot.create(:user)
+    contact.contact_requests << @user
+    expect(contact.contact_requests).to include(@user)
+    expect(contact.contacts).to_not include(@user)
+  end
+
+  it "deletes contact request after denying it" do
```

Comment:
In diesem Test wird nichts denyed. Es wird nur getestet, ob das löschen richtig funktioniert oder?

Reply:
auf das Kreuz drücken bedeutet hier, dass du die Kontakanfrage nicht annimmst, also deniest.

Reply:
Aber hier wird das auf das rote Kreuz drücken doch gar nicht getestet? Und auch nicht eine entsprechende Funktion aufgerufen

Reply:
fair